emcc.py

```diff
@@ -3528,7 +3528,7 @@ def parse_string_list(text):
     # simpler syntaxes
     try:
       return json.loads(text)
-    except:
+    except ValueError:
```

Comment:
Again, how can we be sure this is the only type of exception that can arise here?

Reply:
From [documentation](https://docs.python.org/3/library/json.html#json.loads):

> If the data being deserialized is not a valid JSON document, a `JSONDecodeError` [subclass of `ValueError`] will be raised.

This is the only kind of error we want to recover from.